src/Configuration/InformaticsGatewayConfiguration.cs

```diff
@@ -82,6 +82,12 @@ public class InformaticsGatewayConfiguration
         [ConfigurationKeyName("plugins")]
         public PlugInConfiguration PlugInConfigurations { get; set; }
 
+        /// <summary>
+        /// Represents the <c>endpointSettings</c> section of the configuration file.
+        /// </summary>
+        [ConfigurationKeyName("endpointSettings")]

```

Comment:
Can we move this under `InformaticsGatewayConfiguration` > `HttpConfigurations`?
The reason is that we have many other endpoints DIMSE, HL7, etc...
Thanks!

Reply:
Made this change 👍